src/store/modules/directory.js

```diff
@@ -11,7 +11,7 @@ export const useDirectoryStore = defineStore(
 		actions: {
 			setListingItem(listingItem) {
 				this.listingItem = listingItem && new Listing(listingItem)
-				console.log('Active directory item set to ' + listingItem && listingItem.id)
+				console.log('Active directory item set to ' + listingItem.id)
```

Comment:
Als de listing item naar false wordt gezet, logt de console `Active directory item set to undefined`

Mij lijkt dat een probleem, maar dit is niet zo belangrijk